pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml elements
+
+> Extract elements and display structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements from an XML document (producing XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} > {{path/to/elements}}`
```

Comment:
Is there a typical file extention for those `XPATHs` (like you have `.xml` for XML) or is it just a plain text file?

Reply:
I've seen .xpath used. Thank you.

Reply:
canonic uses ".xpath".
```
❯ grep ".xpath" xml-*.md
xml-canonic.md:`xml canonic --exc-with-comments {{path/to/input.xml|URI}} {{path/to/c14n.xpath}}`
xml-elements.md:`xml elements {{path/to/input.xml|URI}} > {{path/to/elements.xpath}}`
xml-elements.md:`xml elements -a {{path/to/input.xml|URI}} > {{path/to/elements.xpath}}`
xml-elements.md:`xml elements -v {{path/to/input.xml|URI}} > {{path/to/elements.xpath}}`
```